packages/code-du-travail-api/src/server/conf/nlpApi.js

```diff
@@ -0,0 +1,23 @@
+const { logger } = require("../utils/logger");
+const http = require("http");
+
+const NLP_URL = process.env.NLP_URL || "localhost:5000";
+
+logger.info(`nlp-api at ${NLP_URL}`);
+
+module.exports = function quest2keys(query) {
+  const promise = new Promise((resolve, reject) => {
+    http
+      .get(`http://${NLP_URL}/api/quest2keys?q=${query}`, response => {
+        let data = "";
+        response.on("data", chunk => {
+          data += chunk;
+        });
+        response.on("end", () => {
+          resolve(data.toString());
+        });
+      })
+      .on("error", err => reject(err));
+  });
+  return promise;
+};
```

Comment:
pourquoi pas node-fetch ?

Reply:
comme j'avais pas besoin de support json, je suis allé au plus simple. Je voulais un truc le plus léger possible mais c'est probablement négligeable par rapport au temps de la requete.
Tu pense que c'est plus simple / compréhensible avec node-fetch ?

Reply:
just for consistency

Reply:
c'est juste :)